testutils/lint: TestLint failed

testutils/lint.TestLint [failed](https://server-url-not-found-in-env.com/buildConfiguration/Cockroach_UnitTests_Check/6230674?buildTab=log) with [artifacts](https://server-url-not-found-in-env.com/buildConfiguration/Cockroach_UnitTests_Check/6230674?buildTab=artifacts#/) on master @ [ec73c69d203f9018d308e7798b382f289d470092](https://github.com/cockroachdb/cockroach/commits/ec73c69d203f9018d308e7798b382f289d470092):


```
=== RUN   TestLint
--- FAIL: TestLint (1073.75s)
=== RUN   TestLint/TestStaticCheck
    lint_test.go:1773: 
        pkg/sql/ttl/ttljob/ttljob_test.go:423:10: const expiredRowsPerRange is unused (U1000)
    --- FAIL: TestLint/TestStaticCheck (863.73s)
```
<p>Parameters: <code>GOFLAGS=-json</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/test-eng
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-18961

testutils/lint.TestLint [failed](https://server-url-not-found-in-env.com/buildConfiguration/Cockroach_UnitTests_Check/6238314?buildTab=log) with [artifacts](https://server-url-not-found-in-env.com/buildConfiguration/Cockroach_UnitTests_Check/6238314?buildTab=artifacts#/) on master @ [9a7ec4bc51bf5e3f1b469de318c0878470d681fb](https://github.com/cockroachdb/cockroach/commits/9a7ec4bc51bf5e3f1b469de318c0878470d681fb):


```
=== RUN   TestLint
--- FAIL: TestLint (1093.07s)
=== RUN   TestLint/TestStaticCheck
    lint_test.go:1773: 
        pkg/sql/ttl/ttljob/ttljob_test.go:423:10: const expiredRowsPerRange is unused (U1000)
    --- FAIL: TestLint/TestStaticCheck (876.76s)
```
<p>Parameters: <code>GOFLAGS=-json</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/sql/ttl/ttljob/ttljob_test.go
+++ b/pkg/sql/ttl/ttljob/ttljob_test.go
@@ -420,7 +420,6 @@ func TestRowLevelTTLJobMultipleNodes(t *testing.T) {
 				tableName,
 			)
 			const rowsPerRange = 10
-			const expiredRowsPerRange = rowsPerRange / 2
 			splitPoints := make([]serverutils.SplitPoint, len(splitAts))
 			for i, splitAt := range splitAts {
 				newLeaseHolderIdx := (leaseHolderIdx + 1 + i) % numNodes

